src/cli_util.rs

```diff
@@ -1735,7 +1735,8 @@ pub fn parse_args(
     Ok((matches, args))
 }
 
-// TODO: Return std::process::ExitCode instead, once our MSRV is >= 1.61
+// TODO: Return std::process::ExitCode instead, once
+// feature(exitcode_exit_method) is stabilized
```

Comment:
What is the point of `run_and_exit()` in `CliRunner`? Why not have `run()` instead, which returns an `ExitCode`, which can be returned by `main()`?

Reply:
If `run()` doesn't terminate and caller can do more stuff after `run()`, I think `Ui` shouldn't be consumed by `run()`. This means `run(&mut Ui) -> ExitCode` or `run() -> (Ui, ExitCode)`, and we would have to copy some epilogue code.

Reply:
I'm happy to address this in a follow-up change once the discussion concludes. I don't really have an opinion.

I'll resolve this conversation for now and merge the PR. Don't take that as a sign of me ignoring the comments.